security/advancedtls/testdata/1.crl

```diff
@@ -0,0 +1,10 @@
+-----BEGIN X509 CRL-----
```

Comment:
Can we have a sub directory under testdata, and put the CRL-related test certs into it? Also is it possible to document the steps to generate these test files in a README file, like this one: https://github.com/grpc/grpc-go/blob/master/security/advancedtls/testdata/README.md?
That will help us maintain the code in the future. Thank you so much!

Reply:
yes, will do